src/main/java/com/animalfinder/AnimalFinderController.java

```diff
@@ -55,10 +55,9 @@ public ModelAndView loyaltyBreed(Model model) {
 	}
 	
 	@RequestMapping("/searchAnimal")
-	public String searchAnimal(@RequestParam(value="sea"
-			+ "rchTerm", required = false, defaultValue ="") String searchTerm) 
+	public String searchAnimal(@RequestParam(value="searchTerm", required = false, defaultValue ="") String searchTerm) 
 	{
-		String enhancedTerm = searchTerm + "";
+		
 		 try {
 			java.util.List<BreedDTO> fetchAnimals = breedService.fetchAnimals(searchTerm);

```

Comment:
As long as you've removed the java.awt.List import, might as well import java.util.List, so that you don't have to fully qualify list, each time you need it.